### PR TITLE
Added tag links at the top of a blog article

### PR DESCRIPTION
**Pull request by natapg**
Closes WEB-177

### CODE REVIEW

File: src/components/common/Collapsible/Collapsible.module.scss [hunk 1]
```diff
@@ -29,15 +29,11 @@ $side-padding: 2rem;
     font-weight: bold;
 
     &.headerPlus {
-      font-weight: v('normal');
```
Comment (natapg) [inline src/components/common/Collapsible/Collapsible.module.scss:32]: this minor change was requested by Kate today
https://www.figma.com/file/niHrIRowTbqQhDO7W5CPNf/FPJS-WEB-UI?node-id=1863%3A12966

File: src/components/TagList/TagList.module.scss [hunk 2]
```diff
@@ -18,12 +18,12 @@
 }
 
 .item {
-  margin: rem(8px) rem(8px) rem(8px) 0;
+  margin: rem(8px) rem(8px) 0 0;
   padding-bottom: rem(8px);
```
Comment (tan-90) [inline src/components/TagList/TagList.module.scss:22]: Why do we need this to be padding? I assume to prevent margins from collapsing?

**Review by tan-90 — APPROVED** [review]
Looks good, just one small question

File: src/components/TagList/TagList.module.scss (hunk 2 above)
Comment (natapg) [reply src/components/TagList/TagList.module.scss:22]: image

yes !

**Pull request merged**